charts/karmada-operator/templates/karmada-operator-deployment.yaml

```
@@ -37,12 +37,13 @@ spec:
         image: {{ template "karmada.operator.image" . }}
         imagePullPolicy: {{ .Values.operator.image.pullPolicy }}
         ports:
-        - containerPort: 8080
+        - containerPort: {{ .Values.service.port }}
           name: metrics
           protocol: TCP
         command:
         - /bin/karmada-operator
         - --leader-elect-resource-namespace={{ .Release.Namespace }}
```

How about:
```suggestion
        env:
        - name: POD_IP
          valueFrom:
            fieldRef:
              fieldPath: status.podIP
        command:
        - /bin/karmada-operator
        - --leader-elect-resource-namespace={{ .Release.Namespace }}
        - --metrics-bind-address=$(POD_IP):{{ .Values.service.port }}
```
This can avoid All-zero listening. Although there is already an issue #6266 tracking the relevant changes, this is a newly added listening on all zeros, which may be overlooked.

---

Great. 
The following situations also need to be considered
`Set this to "0" to disable the metrics server. `